src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json

```diff
@@ -155,6 +155,10 @@
       "gulfWarStartDate": "1990-09-XX",
       "gulfWarEndDate": "1991-01-XX"
     },
+    "view:toxicExposureDates": {
```

Comment:
This data should match the same month/year format as the Gulf War dates. Let's replace the "day" values with `XX`.